codecov.requirement

```diff
@@ -0,0 +1,5 @@
+Requirements of code coverage percentage for packages
+Use this format: "Package name \t Required percentage"
+istio.io/mixer/adapter/denyChecker	99
```

Comment:
It should be maintained by dev teams i.e. you guys. You can set the thresholds for your pace. What here is just a example showing what format it's supposed to be.